zenoh-plugin-mqtt/src/lib.rs

```diff
@@ -481,9 +485,15 @@ fn treat_admin_query(query: Query, admin_keyexpr_prefix: &keyexpr, config: &Conf
     // send replies
     for (ke, v) in kvs.drain(..) {
         let admin_keyexpr = admin_keyexpr_prefix / ke;
-        use zenoh::prelude::sync::SyncResolve;
-        if let Err(e) = query.reply(Ok(Sample::new(admin_keyexpr, v))).res_sync() {
-            tracing::warn!("Error replying to admin query {:?}: {}", query, e);
+        match zenoh::bytes::Serialize::serialize(ZSerde, v) {
```

Comment:
No need to use ZSerde directly, here `try_into()` works. See https://github.com/eclipse-zenoh/zenoh-plugin-mqtt/pull/131

Reply:
Sure thing.